Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -1208,3 +1211,15 @@ def visit(visitor, obj):
     if obj.version == 1:
         visitor.seen.update(obj.paletteLabels)
         visitor.seen.update(obj.paletteEntryLabels)
+
+
+@NameRecordVisitor.register(
+    (
+        ttLib.getTableClass("cmap"),
+        ttLib.getTableClass("glyf"),
+        ttLib.getTableClass("CFF "),
+        ttLib.getTableClass("CFF2"),
```

Comment:
hm I don't know.. what if a font has a huge/deep table that you don't know about? It feels to me the selection of tables to visit should be opt-in, not opt-out

Reply:
Sounds good. Let me see if I can implement that easily.